docs/src/app/components/pages/components/avatars.jsx

```diff
@@ -38,7 +38,7 @@ export default class AvatarsPage extends React.Component {
           },
           {
             name: 'size',
-            type: 'number',
+            type: 'integer',
```

Comment:
I think that we should stay close to the react PropType. @shaurya947 What do you think?